2002-07-22  Philip Blundell  <philb@gnu.org>

        * sysdeps/unix/sysv/linux/arm/errlist.c: Remove extra weak alias
        definiton of _old_sys_nerr.  Define _old_sys_errlist as strong
        alias.

2002-05-19  Ulrich Drepper  <drepper@redhat.com>

        * sysdeps/unix/sysv/linux/errlist.c: Remove extra weak alias
        definiton of _old_sys_nerr.  Define _old_sys_errlist as strong
        alias.

--- sysdeps/unix/sysv/linux/errlist.c	6 Jul 2001 04:56:12 -0000	1.8
+++ sysdeps/unix/sysv/linux/errlist.c	3 Jun 2002 06:49:30 -0000	1.8.2.1
@@ -1,4 +1,4 @@
-/* Copyright (C) 1998, 2000 Free Software Foundation, Inc.
+/* Copyright (C) 1998, 2000, 2002 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -41,7 +41,7 @@ 
 strong_alias (__old_sys_nerr, _old_sys_nerr);
 compat_symbol (libc, __old_sys_nerr, _sys_nerr, GLIBC_2_0);
 compat_symbol (libc, _old_sys_nerr, sys_nerr, GLIBC_2_0);
-weak_alias (__old_sys_errlist, _old_sys_errlist);
+strong_alias (__old_sys_errlist, _old_sys_errlist);
 compat_symbol (libc, __old_sys_errlist, _sys_errlist, GLIBC_2_0);
 compat_symbol (libc, _old_sys_errlist, sys_errlist, GLIBC_2_0);
 #endif

--- sysdeps/unix/sysv/linux/arm/errlist.c	6 Jul 2001 04:56:13 -0000	1.4
+++ sysdeps/unix/sysv/linux/arm/errlist.c	24 Jul 2002 11:17:01 -0000	1.5
@@ -1,4 +1,4 @@
-/* Copyright (C) 1998, 2000 Free Software Foundation, Inc.
+/* Copyright (C) 1998, 2000, 2002 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -38,10 +38,9 @@ extern const char *const *__old_sys_errl
 const int __old_sys_nerr = OLD_ERRLIST_SIZE;
 
 strong_alias (__old_sys_nerr, _old_sys_nerr);
-weak_alias (__old_sys_nerr, _old_sys_nerr)
 compat_symbol (libc, __old_sys_nerr, _sys_nerr, GLIBC_2_0);
 compat_symbol (libc, _old_sys_nerr, sys_nerr, GLIBC_2_0);
-weak_alias (__old_sys_errlist, _old_sys_errlist);
+strong_alias (__old_sys_errlist, _old_sys_errlist);
 compat_symbol (libc, __old_sys_errlist, _sys_errlist, GLIBC_2_0);
 compat_symbol (libc, _old_sys_errlist, sys_errlist, GLIBC_2_0);
 #endif

